src/Everlution/EnumBundle/DBAL/EnumDBType.php

```diff
@@ -30,16 +30,16 @@ public function getSQLDeclaration(array $fieldDeclaration, AbstractPlatform $pla
     {
         $platform->markDoctrineTypeCommented($this);
 
-        return $platform->getSmallIntTypeDeclarationSQL($fieldDeclaration);
+        return $platform->getVarcharTypeDeclarationSQL($fieldDeclaration);
```

Comment:
why are we changing SamllInt to Varchar? The whole point was to use integer instead of text...

Reply:
Whole point of storing it as integer was wrong as I mentioned originally. Numbers have nothing to do with what they represent. But nothing is stopping you to keep using them.

Reply:
So what will be stored in database? string or integer?

Reply:
string

Reply:
Ok, I am definitely not going to merge functionality which downgrades the performance.